src/components/Picture.jsx

```diff
@@ -1,5 +1,15 @@
-const Picture = () => {
-  return <div>Picture</div>;
+import Image from "/public/example.webp";
```

Comment:
the image should be import from next/image library
(refer to the first line in the code)
https://acm-ucr.github.io/docusaurus/docs/nextjs/next-image


i thought you meant putting the image into src, which is why i said to use public/example.webp.

should be a simple fix